lib/Report/Generator/Table/Cell.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace PhpBench\Report\Generator\Table;
+
+final class Cell
+{
+    /**
+     * @var mixed
+     */
+    private $value;
+
+    /**
+     * @var array<AdditionalValue>
+     */
+    private $secondaryValues;
+
+    /**
+     * @param mixed $value
+     */
+    public function __construct($value, AdditionalValue ...$additionalValues)
+    {
+        $this->value = $value;
+        $this->secondaryValues = $additionalValues;
```

Comment:
Would like to have a single collection of values, but was very hard to refactor.